### PR TITLE
fix: [Image slider] polish behavior

### PR DESCRIPTION
**Pull request by SaraMZoarob**

### CODE REVIEW

File: packages/ui/src/core/image-comparison-slider/image-comparison-slider.props.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export interface ImageComparisonSliderProps extends React.HTMLAttributes<HTMLDiv
   leftImgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
 	rightImgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
 	fallbackPreviewProps?: FallbackPreviewProps,
+	footerProps?: { leftText?: string, rightText?: string, hideFooter?: boolean };
```
Comment (amrelbialy) [inline packages/ui/src/core/image-comparison-slider/image-comparison-slider.props.ts:18]: I see that we only have tree properties here but we destructing restFooterProps.

File: packages/ui/src/core/image-comparison-slider/image-comparison-slider.component.tsx [hunk 2]
```diff
@@ -82,23 +91,33 @@ const ImageComparisonSlider = intrinsicComponent<ImageComparisonSliderProps, HTM
...
-          {thumbIcon}
-        </Styled.Handle>
-        <Styled.LeftImageWrapper ref={topImageRef} {...imgWrapperProps}>
-          <ImagePreviewComponent {...leftImgProps} fallbackPreviewProps={fallbackPreviewProps} />
-        </Styled.LeftImageWrapper>
-
-        <Styled.RightImageWrapper {...imgWrapperProps}>
-          <ImagePreviewComponent {...rightImgProps} fallbackPreviewProps={fallbackPreviewProps} />
-        </Styled.RightImageWrapper>
+        <Styled.SliderWrapper {...rest}>
```
Comment (amrelbialy) [inline packages/ui/src/core/image-comparison-slider/image-comparison-slider.component.tsx:94]: We shouldn't have rest on two elements

File: packages/ui/src/core/image-comparison-slider/image-comparison-slider.styles.ts [hunk 3]
```diff
@@ -117,18 +118,36 @@ const FallbackPreviewWrapper = styled.div.attrs({
...
+const SliderWrapper = styled.div.attrs({
+  className: generateClassNames(baseClassName, 'SliderWrapper'),
+})`
+  position: relative;
+  overflow: hidden;
+  touch-action: none;
+`
+
+const Footer = styled.div.attrs({
+  className: generateClassNames(baseClassName, 'Footer'),
```
Comment (amrelbialy) [inline packages/ui/src/core/image-comparison-slider/image-comparison-slider.styles.ts:139]: all of the classnames should start with small case

File: packages/ui/src/core/image-comparison-slider/image-comparison-slider.props.ts (hunk 1 above)
Comment (SaraMZoarob) [reply packages/ui/src/core/image-comparison-slider/image-comparison-slider.props.ts:18]: because if someone wants to pass any other props to the footer

**Review by amrelbialy — APPROVED** [review]

**Pull request merged**